api/services/Flairs.js

```diff
@@ -63,38 +63,57 @@ exports.hasEventFlair = function(user) {
   return !!flairClasses.match(eventFlairRegExp);
 };
 
-exports.formattedName = function(name) {
-  if (!name) {
+exports.formattedName = function(name, subreddit) {
+  const flairNames = {
+    "pokemontrades": {
+      "default" : "Gen 1 Dex",
+      "gen2" : "Gen 3 Dex",
```

Comment:
We'll probably want to do a flair migration and change the css class name from `gen2` to `gen3` for mod usability.